aspnet/getting-started/installing-on-linux.rst

```diff
@@ -105,22 +105,9 @@ Once this step is complete you should be able to run ``dnvm`` and see some help
 Install the .NET Execution Environment (DNX)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The .NET Execution Environment (DNX) is used to build and run .NET projects. Use DNVM to install DNX for `Mono <http://mono-project.com>`_ or .NET Core (see :doc:`choosing-the-right-dotnet`).
-
-**To install DNX for .NET Core:**
-
-1. Install the DNX prerequisites::
-
-    sudo yum -y install epel-release
-    sudo yum -y install libunwind gettext libcurl-devel openssl-devel zlib
+The .NET Execution Environment (DNX) is used to build and run .NET projects. Use DNVM to install DNX for `Mono <http://mono-project.com>`_ (see :doc:`choosing-the-right-dotnet`).
 
-.. note:: This will add the epel-release repository to your repo list.
-
-2. Use DNVM to install DNX for .NET Core::
-
-    dnvm upgrade -r coreclr
-
-.. note:: .NET Core on Linux is still in early preview. Please refer to the latest `Release Notes <https://github.com/aspnet/home/releases>`__ for known issues and limitations.
+.. note:: DNX support for .NET Core is not available for CentOS, Fedora and derivative in this release, but will be enabled in a future release.
```

Comment:
When it doesn't end up working, because they have any other version of ICU do we expect to respond to issues they open?  Feels like it will be very noisy.